test/helpers/utils.js

```diff
@@ -15,214 +24,168 @@ async function getUserToken(username, password) {
-              Authorization: `Basic ${Buffer.from(`${username}:${password}`).toString('base64')}`,
-              'Content-Type': 'application/x-www-form-urlencoded'
-            }
-          },
-          function (error, response, body) {
-                statusCode = response.statusCode;
-          }
-      ).catch(error => {
-        console.log(error);
-      });
-      if(retryCount > 1) {
-        logger.info("retrying idam code response " + retryCount);
-      }
-      retryCount++;
-  } while (retryCount <= 3 && statusCode > 300);
-  const code = JSON.parse(codeResponse).code;
-
-  const idamAuthPath = `/oauth2/token?grant_type=authorization_code&client_id=divorce&client_secret=${idamClientSecret}&redirect_uri=${redirectUri}&code=${code}`;
-  var authTokenResponse;
-  retryCount = 0;
-  statusCode = 400;
-  do {
-      authTokenResponse = await request.post({
-          uri: idamBaseUrl + idamAuthPath,
-            headers: { 'Content-Type': 'application/x-www-form-urlencoded' }
-          },
-         function (error, response, body) {
-               statusCode = response.statusCode;
-         }
-      ).catch(error => {
-               console.log(error);
-             });
-      if(retryCount > 1) {
-              logger.info("retrying idam token response " + retryCount);
-      }
-      retryCount++;
-  } while (retryCount <= 3 && statusCode > 300);
-
-  logger.debug(JSON.parse(authTokenResponse).access_token);
-
-  return JSON.parse(authTokenResponse).access_token;
+  const idamCodeResponse = await axiosRequest({
+    method: 'post',
+    url: idamBaseUrl + idamCodePath,
+    headers: {
+      Authorization: `Basic ${Buffer.from(`${username}:${password}`).toString('base64')}`,
+      'Content-Type': 'application/x-www-form-urlencoded'
+  }});
+
+  logger.info("Successfully retired idam code")
+
+  const idamAuthPath = `/oauth2/token?grant_type=authorization_code&client_id=divorce&client_secret=${idamClientSecret}&redirect_uri=${redirectUri}&code=${idamCodeResponse.data.code}`;
+
+  const authTokenResponse = await axiosRequest({
+    method: 'post',
+    url: idamBaseUrl + idamAuthPath,
+    headers: {
+      'Content-Type': 'application/x-www-form-urlencoded'
+    }
+  });
+
+  logger.info("Successfully retired user token");
+
+  return authTokenResponse.data.access_token;
 }
 
 async function getUserId(authToken) {
   logger.info('Getting User Id');
 
   const idamBaseUrl = `https://idam-api.${env}.platform.hmcts.net`;
-
   const idamDetailsPath = '/details';
-  const userDetails = await request.get({
-    uri: idamBaseUrl + idamDetailsPath,
+
+  const userDetailsResponse = await axiosRequest({
+    method: 'get',
+    url: idamBaseUrl + idamDetailsPath,
     headers: { Authorization: `Bearer ${authToken}` }
   });
 
-  logger.debug(JSON.parse(userDetails).id);
-
-  return JSON.parse(userDetails).id;
+  logger.info("Successfully retrieved User ID: ", userDetailsResponse.data.id);
+  return userDetailsResponse.data.id;
```

Comment:
Just checking line 64 is fine.  Everywhere else you use %s style string interpolation.

Reply:
(you may have just forgotten the %s)